src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -57,11 +57,15 @@ public static class TextCatalog
                 outputColumnName, inputColumnNames, options);
 
         /// <summary>
-        /// Tokenize incoming text in <paramref name="inputColumnName"/> and output the tokens as <paramref name="outputColumnName"/>.
+        /// Create a <see cref="TokenizingByCharactersEstimator"/>, which tokenizes characters by splitting text into sequences of characters
+        /// using a sliding window.
         /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be a vector of keys.</param>
```

Comment:
>vector [](start = 46, length = 6)

unknown size vector

Reply:
Is variable-sized vector  fine?


---
In reply to: [276869836](https://github.com/dotnet/machinelearning/pull/3418#discussion_r276869836) [](ancestors = 276869836)